### ISSUE TITLE
Perioral hyperkeratosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6808]
**Preferred term label:**
Perioral hyperkeratosis

**Synonyms**
Periorifical hyperkeratosis

**Definition (free text, please give PubMed ID)**
Increased thickness of the stratum corneum (the outer layer of the skin) in the skin surrounding the mouth.
 PMID:32965877 

**Parent term (use hpo.jax.org/app)**
Hyperkeratosis HP:0000962

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619208 
 PMID: 30321533  4/4
PMID: 31361044 1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
